src/include/platform/internal/GenericConfigurationManagerImpl.cpp

```diff
@@ -794,7 +794,7 @@ CHIP_ERROR GenericConfigurationManagerImpl<ImplClass>::_ClearServiceProvisioning
         ChipDeviceEvent event;
         event.Type                                   = DeviceEventType::kAccountPairingChange;
         event.AccountPairingChange.IsPairedToAccount = false;
-        PlatformMgr().PostEvent(&event);
+        ReturnErrorOnFailure(PlatformMgr().PostEvent(&event));
```

Comment:
If we take this early return, we have done the `ClearConfigValue` calls above but not the `mFlags.Clear` calls below.  Is that an OK state to be in?  Or do we make assumptions based on `mFlags` about the config value bits?

At the very least could use documentation.  Same for the other early return here.

Reply:
Will change to continue and return an error at the end, and add to the existing TODO comment.